src/components/DayPicker.jsx

```diff
@@ -365,14 +369,19 @@ export default class DayPicker extends React.Component {
   }
 
   onPrevMonthClick(nextFocusedDate, e) {
+    const { translationValueRTL } = this.state;
+    const { isRTL } = this.props;
+
     if (e) e.preventDefault();
 
     if (this.props.onPrevMonthClick) {
       this.props.onPrevMonthClick(e);
     }
 
-    const translationValue =
-      this.isVertical() ? this.getMonthHeightByIndex(0) : this.dayPickerWidth;
+    let translationValue = this.isVertical() ? this.getMonthHeightByIndex(0) : this.dayPickerWidth;
```

Comment:
yeah remember i had to change it because of the "maximum line length of 100" lint rule. it exceeds it by 2 chars. 
The difference between this and the other block from `onPrevMonthClick` is the `-` chars on this line:
`let translationValue = this.isVertical() ? -this.getMonthHeightByIndex(1) : -this.dayPickerWidth;`
should i disable this rule for this line only?